### ISSUE TITLE
TOS contains lists for stylization purposes only

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3979]
## Area or Component
Patient Experience - Terms of Service

## Quick Summary
There are several non-list elements contained in a list on the terms of service. "User", "Entity", etc are more appropriately subheadings. 

## Accessibility Violation
Two:
`<ul> and <ol> must only directly contain <li>, <script> or <template> elements.` https://dequeuniversity.com/rules/axe/4.4/list?application=AxeChrome

`Content is not a list but is marked as such`
https://axe.deque.com/rule-help/en/semantic-list-used-for-presentation
## Action Requested
See comments below for reference. The changes requested include:
- Consolidate list items and paragraph text in "Definitions," and bold defined words
- Change list items to subheadings in "Data rights and usage"
- Make "Accounts/registration" sentence case for consistency
- Switch user types to subheadings in "Other responsibilities," and change indentation of list items underneath 

Note: These changes were approved by the CDC’s Office of General Counsel through Mike Judd.

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/lang/en.ts#L368-L368

## Original Deque Issue
list | Ensures that lists are structured correctly | Non-list element contained the Terms of Service ("user", "entity" etc are more appropriately subheadings) | <ul> and <ol> must only directly contain <li>, <script> or <template> elements | https://dequeuniversity.com/rules/axe/4.4/list?application=AxeChrome | serious | No | No |   | ["ul:nth-child(27)"] | Fix all of the following:  List element has direct children that are not allowed inside <li> elements | <ul> | cat.structure,wcag2a,wcag131 | rnv7@cdc.gov | Patient Self Reg - Terms of Service | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T20:53:04.526Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by lauraykerr** [comment]
@emmastephenson @fzhao99 Can you confirm that what's in the [source code linked here](https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/lang/en.ts#L368-L368) is what's live on the site TOS? I think I'm seeing a couple discrepancies but I'm not sure.

**Comment by emmastephenson** [comment]
@lauraykerr I believe that is the source of truth of the TOS, yes! There is also the terms of service listed on the [static site](https://www.simplereport.gov/terms-of-service/), which may have gotten slightly out of sync with the app version. If that's true, we'll just make sure to align them whenever we go and make the TOS changes.

**Comment by lauraykerr** [comment]
> ### Definitions
> 
> - **Entity**: A health care provider or facility; testing site; a state, local, tribal, and territorial public health authority...
> - **User**: An individual whose personal data is being reported via Simple Report (Individual User)...
> 

Explanation: Keep "Entity" and "User" as list items since this is a definitions list, but move definitions into list items of the words they define.

**Comment by lauraykerr** [comment]
> ### Data rights and usage 
> #### Accounts/registration
> ##### For entity users
> If you are using the Application on behalf of an Entity....
> ##### For entity administrators
> The Entity Administrator agrees to verify the identity of other Entity Users...
> ##### For individual users
> Entity Administrators will grant Individual Users access to the Application...
Explanation: Change list items to subheadings, and make "Accounts/registration.

**Comment by lauraykerr** [comment]
> ### Other responsibilities 
> #### For all users
> - You will be fully accountable for all the data...
> - You will promptly inform CDC in the event...
> #### For entity administrators and entity users
> - You must adhere to the basic desktop security measures...
> - As may be required by applicable law....

Explanation: Switch user types to subheadings and make list items underneath top level.

**Comment by lauraykerr** [comment]
@emmastephenson Reaching out to Mike Judd for guidance on legal review for these and the other TOS updates.

**Comment by emmastephenson** [comment]
Perfect, thank you @lauraykerr!

**Comment by lauraykerr** [comment]
@zdeveloper The content changes (outlined in the description and comments) have legal approval and are ready for development

**Comment by lauraykerr** [comment]
[#353](https://github.com/CDCgov/prime-simplereport-site/pull/353) resolves this issue for static site TOS.
